### PR TITLE
Line list medication fix

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** https://www.pivotaltracker.com/story/show/173123723

## Because

We should only show patient's current medications in the line list

### CODE REVIEW

File: app/exporters/patients_exporter.rb [hunk 1]
```diff
@@ -119,6 +119,6 @@ def self.csv_fields(patient)
   end
 
   def self.medications_for(patient)
-    patient.prescription_drugs.flat_map { |drug| [drug.name, drug.dosage] }
+    patient.current_prescription_drugs.flat_map { |drug| [drug.name, drug.dosage] }
```
Comment (kitallis) [inline app/exporters/patients_exporter.rb:122]: This scope, of course.

Comment (harimohanraj89) [reply app/exporters/patients_exporter.rb:122]: I know. I think we assumed initially that the `deleted_at` flag was being set on deleted prescription drugs, in which case the default scope would exclude deleted drugs. Now we know this isn't the case.

Comment (kitallis) [reply app/exporters/patients_exporter.rb:122]: Yeah, maybe during the medication history stuff, we could see if we can move this to be `deleted_at` and not this special `is_deleted` flag.

Comment (harimohanraj89) [reply app/exporters/patients_exporter.rb:122]: Yup, we've listed it as a "nice-to-do-while-we're-here" 👍

**Pull request merged**